modules/gke-cluster-autopilot/variables.tf

```diff
@@ -34,6 +34,12 @@ variable "backup_configs" {
   nullable = false
 }
 
+variable "deletion_protection" {
+  description = "Whether or not to allow Terraform to destroy the cluster. Unless this field is set to false in Terraform state, a terraform destroy or terraform apply that would delete the cluster will fail."
+  default     = true
```

Comment:
we should have it as false by default Julio

Reply:
the provider defaults to true

Reply:
well the provider is not always sane :) anyway, we discussed it and I understasnd your point, I would just do the same change on GCS

Reply:
GCS already has it, I stand corrected; can we bubble up this variable in GKE blueprints too? anf FAST

Reply:
As discussed, I'll update `cloudsql-instance` which is behaving in the opposite way and expose it in blueprints to allow easy destruction.